data/json/construction.json

```diff
@@ -7867,5 +7867,62 @@
     "components": [ [ [ "rock_large", 22 ] ] ],
     "pre_terrain": "t_dirt",
     "post_terrain": "t_rockyobstacle"
+  },
+  {
+    "type": "construction",
+    "id": "constr_brick_oven_prep",
+    "group": "build_brick_oven",
+    "category": "CONSTRUCT",
+    "required_skills": [ [ "fabrication", 0 ] ],
+    "time": "2 h",
+    "tools": [ [ [ "shovel", -1 ], [ "primitive_shovel", -1 ] ] ],
+    "qualities": [ [ { "id": "HAMMER", "level": 2 } ] ],
+    "components": [ [ [ "fire_brick", 30 ] ], [ [ "clay_lump", 5 ] ], [ [ "water", 2 ], [ "water_clean", 2 ] ] ],
```

Comment:
There's no requirement already made that includes both clean and regular water you could use there instead of listing both? If not it's probably worth it to make that requirement